### PR TITLE
LPS-107703 Adapt test

### PR DESCRIPTION
**Pull request by ruben-pulido**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:test:relevant

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 1]
```diff
@@ -949,6 +956,92 @@ components:
             required:
                 - name
             type: object
+        FragmentContentField:
+            properties:
+                id:
+                    type: string
+                link:
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: this has to be fragmentLink, if the property links to a schema, should be called the same.

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 2]
```diff
@@ -949,6 +956,92 @@ components:
...
+                value:
+                    oneOf:
+                        - $ref: "#/components/schemas/FragmentContentFieldBackgroundImage"
+                        - $ref: "#/components/schemas/FragmentContentFieldHTML"
+                        - $ref: "#/components/schemas/FragmentContentFieldImage"
+                        - $ref: "#/components/schemas/FragmentContentFieldText"
+            type: object
+        FragmentContentFieldBackgroundImage:
+            properties:
+                backgroundImage:
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: let's hope this pass

File: modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml [hunk 3]
```diff
@@ -949,6 +956,92 @@ components:
...
+        FragmentContentFieldHTML:
+            properties:
+                html:
+                    oneOf:
+                        - $ref: "#/components/schemas/InlineValue"
+                        - $ref: "#/components/schemas/MappedValue"
+            type: object
+        FragmentContentFieldImage:
+            properties:
+                image:
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml]: fragmentImage

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 4]
```diff
@@ -0,0 +1,582 @@
...
+					put(key, jsonObject.get(key));
+				}
+			}
+		};
+	}
+
+	private MasterPage _toMasterPage(long masterLayoutPlid, Locale locale) {
+		Layout layout = _layoutLocalService.fetchLayout(masterLayoutPlid);
+
+		if (layout != null) {
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: in this case he prefers the setName(() => and then check nullity inside

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 5]
```diff
@@ -0,0 +1,582 @@
...
+					type = PageElement.Type.ROW;
+				}
+			};
+		}
+
+		return null;
+	}
+
+	private PageElement[] _toPageElements(
+			LayoutPageTemplateEntry layoutPageTemplateEntry, Layout layout,
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: sort

File: modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java [hunk 6]
```diff
@@ -0,0 +1,582 @@
...
+				setPaddingTop(
+					() -> {
+						if (configJSONObject.isNull("paddingTop")) {
+							return null;
+						}
+
+						return configJSONObject.getInt("paddingTop");
+					});
+
+				containerType = ContainerType.valueOf(
```
Comment (nhpatt) [inline modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/PageDefinitionDTOConverter.java]: this should be before the setters

**Pull request closed without merging**